frontend/src/components/home/MeasurementsAccordion.tsx

```diff
@@ -0,0 +1,39 @@
+import { t } from "../utils";
+import { Accordion } from "react-bootstrap";
+import ProjectSection from "./sections/ProjectSection";
+import ClientSection from "./sections/ClientSection";
+import InvoiceSection from "./sections/InvoiceSection";
+
+const MeasurementsAccordion = () => {
+  return (
+    <Accordion defaultActiveKey="0">
+      <Accordion.Item eventKey="0">
+        <Accordion.Header>
+          <h3>{t("nav.projects")}</h3>
+        </Accordion.Header>
+        <Accordion.Body>
+          <ProjectSection />
+        </Accordion.Body>
+      </Accordion.Item>
+      <Accordion.Item eventKey="1">
+        <Accordion.Header>
+          <h3>{t("nav.clients")}</h3>
+        </Accordion.Header>
+        <Accordion.Body>
+          <ClientSection />
+        </Accordion.Body>
+      </Accordion.Item>
+      <Accordion.Item eventKey="2">
+        <Accordion.Header>
+          <h3>{t("nav.invoices")}</h3>
+        </Accordion.Header>
+        <Accordion.Body>
+          <InvoiceSection />
+        </Accordion.Body>
+      </Accordion.Item>
+    </Accordion>
+  );
+};
+
+export default MeasurementsAccordion;
+
```

Comment:
Fyi:

Ik vind het persoonlijk makkelijker om met named exports te werken:
- Je hebt de export default op het einde niet meer (=2 lijnen code minder per file)
- Ik heb het gevoel dat intellisense soms beter is bij named exports (vb bij automatic importing)
- De named import heeft altijd dezelfde naam -- bij default imports & file/function renames heb je meer refactor werk bij default exports